src/patterns/question-pages/index.md.njk

```diff
@@ -83,6 +83,19 @@ For example, ‘About you’
 
 You can also learn more about how starting with [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page) helps users in the GOV.UK Service Manual.
 
+#### Asking complex questions without using hint text
+
+Do not use hint text if you need to give a lengthy explanation with lists and paragraphs. Screen readers read out the entire text when users interact with the form element. This could frustrate users if the text is long.
+
+If you're asking a question that needs a detailed explanation, use:
+* a `h1` heading that's a statement (for example, 'Add undergraduate degree') rather than a question
+* whatever mix of text, paragraphs, lists and examples best explains your question to users
+* a label, above the form input, that asks users a specific question – for example, 'What type of degree do you want to add?'
+
+Where possible, you should also make sure your `h1` heading is consistent with your service's other h1 headings.
+
+{{ example({group: "patterns", item: "question-pages", example: "explanatory-text", html: true, open: true, hideTab:true}) }}
```

Comment:
This example is currently configured to have the HTML code always visible, which takes up quite a lot of space (see https://deploy-preview-1716--govuk-design-system-preview.netlify.app/patterns/question-pages/#asking-complex-questions-without-using-hint-text).

Is there a particular reason to go with this configuration? Or can we go with the normal config of having tabs and the code hidden by default?

```suggestion
{{ example({group: "patterns", item: "question-pages", example: "explanatory-text", html: true, nunjucks: true, open: false}) }}
```

Reply:
Thanks, @lfdebrux! Looks good to me and @christopherthomasdesign.